internal/logger/target/http/http.go

```diff
@@ -105,23 +108,29 @@ type Target struct {
 	logCh   chan interface{}
 	logChMu sync.RWMutex
 
+	// If this webhook is being re-configured we will
+	// assign the new webhook target to this field.
+	// The Send() method will then re-direct entries
+	// to the new target when the current on has
+	// been seet to status "statusClosed".
+	// Once the glogal webhook slice has been update
+	// the current target will stop receiving entries.
+	migrateTarget *Target
```

Comment:
I don't see this being assigned to a target. Has this PR moved away from this approach?

Reply:
yep, you're right.. the code for that was located in a previous global buffer migration method, which was changed. It's re-implemented.